client/java/src/main/java/com/linecorp/centraldogma/client/AbstractCentralDogmaBuilder.java

```diff
@@ -214,53 +224,65 @@ public final B profile(ClassLoader classLoader, Iterable<String> profiles) {
         checkState(selectedProfile == null, "profile cannot be loaded more than once.");
         checkState(hosts.isEmpty(), "profile() and host() cannot be used together.");
 
-        String selectedProfile = null;
-        ImmutableSet<InetSocketAddress> hosts = null;
-        boolean profilesIsEmpty = true;
-        for (String p : profiles) {
-            checkNotNull(p, "profiles contains null: %s", profiles);
-            profilesIsEmpty = false;
+        final URL resourceUrl = findFirstResource(
```

Comment:
Just curious. What if we don't use `TEST_PROFILE_RESOURCE_PATH`? Or, how about adding a new method to this builder which sets `profileResourcePath` so that a user can specify his or her resource?

Reply:
If `centraldogma-profiles-test.json` is missing, it will automatically fall back into `centraldogma-profiles.json`, just like Logback does for `logback-test.xml` and `logback.xml`. I believe `-test.*` has precedence so that a user puts `-test.*` into `src/test/resources` so that it does not pick up `src/main/resources/`.

I'm curious if we need an option to specify the `InputStream`, `URL` or `File` along with `profile()`. Do we?

Reply:
Added `profileResources(String... paths)` to the builder class.

Reply:
I can see now what you intend. :-) I was just curious what if we provide a way to specify user resource path instead of using a fixed path of `centraldogma-profiles.json` and `centraldogma-profiles-test.json`. But your way is reasonable to me, so please forget my previous comment. Thanks!